ci/push_images

```diff
@@ -12,12 +12,12 @@ badusage=64
 
 nprocs="${1:-1}"
 
-if [[ $TRAVIS_PULL_REQUEST != "false" ]] || [[ $TRAVIS_BRANCH != "develop" ]] ; then
+if [[ ${CURRENT_BRANCH} != "develop" ]] ; then
     echo "PR is not merged to develop, not pushing to DockerHub"
     exit 0;
 fi
 
-echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+echo "${DOCKERHUB_PASSWORD}" | docker login -u "${DOCKERHUB_USER_NAME}" --password-stdin
```

Comment:
You could move this to CI configs. Even better you can use docker actions to login

Reply:
I think we should keep this to be able to use this scripts manually. It does not seem right to me to break the consistency in the script. Docker login is the part of the logic that we packed into script. Either we should remove all the commands and put them in GH Actions or keep the script as is

Reply:
In my local machine, I would not want to store my username and password in env vars that can be compromised.

Docker login should be decoupled with these scripts, or they should be optional.

However, I am ok with this as long as you open a new issue and track this request.

Reply:
https://github.com/citusdata/packaging/issues/828